web/impact/impact/permissions.py

```diff
@@ -58,7 +62,7 @@ def has_permission(self, request, view):
         model_name = model_name_case(model, related_model)
         app_label = 'mc'
         kwargs = {'app': app_label, 'model_name': model_name}
-        perms = [perm % kwargs for perm in self.perms_map[request.method]]
+        perms = [perm % kwargs for perm in method_to_perm(request.method)]
```

Comment:
The list comp, coupled with `method_to_perm` always returning a single-item list, seems unnecessary.

Reply:
Good point.  Again I was just trying not to change the existing behavior, but that one seems even less controversial.